src/utils/.vimrc

```diff
@@ -0,0 +1 @@
+set noexpandtab
```

Comment:
Huh?

Reply:
Not sure why it's committed. But it tells vim to use tab characters.